packages/wonder-blocks-icon/src/util/icon-assets.test.js

```diff
@@ -2,13 +2,16 @@
 describe("icons", () => {
     test("default contains icons we expect", async () => {
         // Arrange
-        const importedModule = import("./icon-assets.js");
+        const importedModule = await import("./icon-assets.js");
```

Comment:
**nit:** This should be in the `// Act` - an empty `// Arrange` is ok, but an empty `// Act` or `// Assert` is a smell, I think.

Reply:
I forgot to do this.  I'll do it just in a abit.